Files log4cl.orig/appender.fasl and log4cl/appender.fasl differ
Common subdirectories: log4cl.orig/debian and log4cl/debian
Files log4cl.orig/layout.fasl and log4cl/layout.fasl differ
diff -u log4cl.orig/layout.lisp log4cl/layout.lisp
--- log4cl.orig/layout.lisp	2004-03-01 12:35:38.000000000 -0600
+++ log4cl/layout.lisp	2005-04-20 21:48:29.000000000 -0500
@@ -132,7 +132,7 @@
    (line-number :initarg :line :initform nil)))
 
 
-(defconstant +colors+ 
+(cl+:defconst +colors+ 
   '(("debug" . "#339933")
     ("info" . "#000000")
     ("warning" . "#993300")
Only in log4cl: layout.lisp~
diff -u log4cl.orig/log4cl.asd log4cl/log4cl.asd
--- log4cl.orig/log4cl.asd	2004-04-24 04:26:55.000000000 -0500
+++ log4cl/log4cl.asd	2005-04-21 18:01:38.000000000 -0500
@@ -19,7 +19,7 @@
 
 
 (defpackage #:log4cl-system 
-  (:use #:cl 
+  (:use #:cl
 	#:asdf))
 
 (in-package #:log4cl-system)
@@ -43,7 +43,8 @@
 	       (:file "appender" :depends-on ("package" "tools"))
 	       (:file "layout" :depends-on ("package" "tools"))
 	       (:file "package-logger" :depends-on ("logger"))
-               (:file "tools" :depends-on ("package"))))
+               (:file "tools" :depends-on ("package")))
+  :depends-on (:cl-plus))
 
 (defsystem :log4cl.syslog
   :components ((:file "appender-syslog" :depends-on ("cl-syslog"))
@@ -52,7 +53,7 @@
 
 (defsystem :log4cl.db
   :components ((:file "appender-db"))
-  :depends-on (:log4cl))
+  :depends-on (:log4cl :clsql))
 
 (defsystem :log4cl.mail
   :components ((:file "appender-mail"))
Only in log4cl: log4cl.asd~
Files log4cl.orig/logger.fasl and log4cl/logger.fasl differ
Files log4cl.orig/package-logger.fasl and log4cl/package-logger.fasl differ
Files log4cl.orig/package.fasl and log4cl/package.fasl differ
Files log4cl.orig/tools.fasl and log4cl/tools.fasl differ
